### ISSUE TITLE
关于原型网络的代码

### ISSUE THREAD

**Issue by khk-abc** [sysulic/FL-MSRE#3]
作者您好，不好意思打扰您。在阅读您FL-MSRE: A Few-Shot Learning based Approach to Multimodal Social Relation Extraction的代码时，发现在models文件夹下的proto文件中，forward中计算logits和pred为什么要进行减1并且使用max呢？         
        **logits = self.__batch_dist__(support, query) # (B, total_Q, N)
        minn, _ = logits.min(-1)  
        logits = torch.cat([logits, minn.unsqueeze(2) - 1], 2) # (B, total_Q, N + 1)
        _, pred = torch.max(logits.view(-1, N+1), 1)**
